app/_kong_plugins/ai-semantic-cache/examples/mistral.yaml

```diff
@@ -0,0 +1,38 @@
+title: 'Set up with Mistral'
+
+description: 'Enable the plugin with Mistral as your LLM and a Redis vector database.'
+
+weight: 900
+
+# requirements: <- not required
+# - "some req"
+
+variables:
+  header_value:
+    value: $MISTRAL_API_KEY
```

Comment:
we now support descriptions for these variables too :). They'll get rendered next to the variable's name

Reply:
```
variables:
  header_value:
    value: $MISTRAL_API_KEY
    description: Some description explaining what it is
```